.github/workflows/tests.yml

```diff
@@ -1,40 +1,40 @@
----
-name: all_green
-
+name: CI
 concurrency:
-  group: ${{ github.head_ref }}
+  group: ${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 on: # yamllint disable-line rule:truthy
   pull_request:
-    types:
-      - opened
-      - reopened
-      - synchronize
     branches:
       - main
       - stable-*
-    tags:
-      - "*"
+  workflow_dispatch:
 
 jobs:
   linters:
     uses: ./.github/workflows/linters.yml # use the callable linters job to run tests
+  changelog:
+    uses: ansible/ansible-content-actions/.github/workflows/changelog.yaml@main
+    if: github.event_name == 'pull_request'
   sanity:
-    uses: ./.github/workflows/sanity.yml # use the callable sanity job to run tests
+    uses: ansible/ansible-content-actions/.github/workflows/sanity.yaml@main
   units:
-    uses: ./.github/workflows/units.yml # use the callable units job to run tests
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
```

Comment:
Does the unit workflow from ansible-content-actions not do what we need?

Reply:
[ansible-content-actions](https://github.com/ansible/ansible-content-actions/blob/main/README.md#scope) run the unit tests on the galaxy version of the collection. It uses [tox-ansible](https://github.com/ansible/tox-ansible) which lacks the capabilities to do a source install.

Reply:
Are we sure about this? When I run tox-ansible locally it looks to me like it's using my local copy. The way I read the docs for the action is that it installs the collection's dependencies from galaxy.

Reply:
I am reaching out to the devtools team regarding this issue. It wasn't functioning earlier, and I'm not sure if there have been any alterations in tox-ansible. Currently, there isn't a workflow in ansible-content-actions to execute unit tests on the source.

Reply:
You're correct. Sorry for the confusion. tox-ansible pulls dependencies from Galaxy, not the source.